features/old/messages/provider_side.feature

```diff
@@ -20,19 +20,11 @@ Feature: Provider side messages
   Scenario: Sending a message
     Given account "bob" has no messages
 
-    And I go to the provider dashboard
-    And I navigate to the accounts page
-    And I follow "bob"
-    And I follow "Send message"
-    Then the "To" field should be fixed to "bob"
+    And I am on the outbox compose page
```

Comment:
I'm afraid this "fix" is not the right one :confused: 
The original test was validating sending messages for individual account, it's done in a pop-up, not the outbox compose page.
In fact, by failing this test caught a real bug (that I introduced :upside_down_face:  ), see the screenshot - a successful message sending got a "null" flash message.

See the top-right corner of the screenshot.
![Screenshot from 2022-10-18 17-53-55](https://user-images.githubusercontent.com/1270328/196548320-aec37a98-ac4f-4df3-8301-8842414d01f1.png)

Reply:
@mayorova which user is this? can you share ur?

yes screenshots looks right, I checked your fix also looks good

Reply:
you reverted the test and I think its same as previous only message is changed which is assigned through variable

Reply:
@nidhi-soni1104 That happens for any user.

Yes, I reverted the test, because it's actually valid.

Indeed, I put the flash message in an instance variable. The reason is that in on this send message form (which is a pop-up), the flash messages are handled differently, and they are expected to be in a variable - no actual reload takes place on that page, it's done through JavaScript.

Reply:
Oh good to  know this